Mathlib/RingTheory/Bialgebra.lean

```diff
@@ -0,0 +1,83 @@
+/-
+Copyright (c) 2024 Ali Ramsey. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ali Ramsey
+-/
+import Mathlib.RingTheory.Coalgebra
+import Mathlib.RingTheory.TensorProduct
```

Comment:
```suggestion
import Mathlib.RingTheory.TensorProduct
```
It seems unlikely that this import is required here.

Reply:
I thought so too -- but it's needed! For coalgebras we only need tensor products of R-modules; for bialgebras we need to claim that comultiplication is a ring homomorphism so we need tensor products of R-algebras.